locmapper/action.yml

```diff
@@ -0,0 +1,11 @@
+name: 'LocMapper'
+description: GitHub Action for using locmapper cli
+inputs:
+  command:
+    description: 'Default input command'
+    required: true
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+  args:
+    - ${{ inputs.command }}
```

Comment:
ags are passed on entrypoints.sh argument. See the docs: https://github.com/happn-tech/LocMapper/blob/main/Dockerfile#L49 . 

I think it needed to create an entrypoint.sh file. Somethink like in the 

`USER root

ADD entrypoint.sh .

RUN chmod +x entrypoint.sh && chown locmapper entrypoint.sh

ENTRYPOINT ["/locmapper/entrypoint.sh"]`

and entrypoint.sh :

`#!/bin/sh

locmapper $1`